src/g_game.c

```diff
@@ -2541,14 +2547,8 @@ static void DoSaveGame(char *name)
   // save max_kill_requirement
   saveg_write32(max_kill_requirement);
 
-  saveg_write8(pistolstart);
-  saveg_write8(coopspawns);
-  saveg_write8(halfplayerdamage);
-  saveg_write8(doubleammo);
-  saveg_write8(aggromonsters);
-
   // [FG] save snapshot
-  saveg_buffer_size(MN_SnapshotDataSize());
+  saveg_grow(MN_SnapshotDataSize());
   MN_WriteSnapshot(save_p);
   save_p += MN_SnapshotDataSize();
```

Comment:
Because of the snapshots, we can't write new options at the end of the file, so the previous approach was incorrect. Maybe it's for the better.